### ISSUE TITLE
can mashr be used in interaction terms

### ISSUE THREAD

**Issue by arkyl** [stephenslab/mashr#109]
Hi,
We were estimating association difference in a variable, by interaction term in linear model, in multiple tissues. I am wondering if the mashr can be used in beta and sd of the interaction term, for meta analysis of association difference in multiple tissues. Thanks a lot!
Yue

**Comment by pcarbo** [comment]
@arkyl Yes, if you have estimates (and se's) of the interaction effect in multiple tissues mashr should be suitable for this analysis. (I'm not quite sure about the meta analysis part.)

**Comment by arkyl** [comment]
Thanks very much for the quick reply! The meta analysis here simply means mashr analysis.
